ftw/subsite/content/subsite.py

```diff
@@ -85,7 +85,32 @@
         name="FromEmail",
         widget=atapi.StringWidget(
             label=_(u'label_fromemail', default=u'Email Senderaddress'),
-            description=_(u'help_fromname', default=u'')))
+            description=_(u'help_fromname', default=u''))),
+
+    atapi.TextField(
+        name='webstatsAnalyticsSnippet',
+        schemata='subsite',
+        default_content_type='text/html',
+        allowable_content_types=('text/html',),
+        validators=('isTidyHtmlWithCleanup', ),
```

Comment:
:+1: For custom permission (Managers only by default)
The plone site itself features the same field, without any code cleanup. The user which enters the tracking code, should know what he is doing.